ZConfig/tests/test_schemaless.py

```diff
@@ -18,7 +18,17 @@
 __docformat__ = "reStructuredText"
 
 import doctest
+import unittest
 
+from ZConfig.schemaless import Section
+
+class TestSection(unittest.TestCase):
+
+    def test_init_with_data(self):
+        s = Section(data={'k': 'v'})
+        self.assertDictEqual(s, {'k': 'v'})
 
 def test_suite():
-    return doctest.DocFileSuite("schemaless.txt", package="ZConfig")
+    suite = unittest.makeSuite(TestSection)
+    suite.addTest(doctest.DocFileSuite("schemaless.txt", package="ZConfig"))
+    return suite
```

Comment:
If you switch to using `defaultTestLoader` everywhere, a nice way to mix it with doctests is
```python
def test_suite():
    return unittest.TestSuite([
        unittest.defaultTestLoader.loadTestsFromName(__name__),
        doctest.DocFileSuite(...),
    ])
```